# Assurance Monitor confidence message with header
std_msgs/Header header
AssuranceMonitorConfidence data